xla/service/gpu/ir_emitter_triton_test.cc

```diff
@@ -69,13 +70,36 @@ namespace {
 namespace m = ::xla::match;
 
 class TritonTest : public GpuCodegenTest {
+  const auto& device_desc() {
+    return backend()
+        .default_stream_executor()
+        ->GetDeviceDescription();
+  }
  public:
   se::CudaComputeCapability GetCudaComputeCapability() {
     return backend()
         .default_stream_executor()
         ->GetDeviceDescription()
         .cuda_compute_capability();
   }
+  const se::GpuComputeCapability& GpuComputeComp() {
+    return device_desc().gpu_compute_capability();
+  }
+  bool SkipBF16Tests() {
```

Comment:
nit: there should be one empty line between definition. I'll make the edit on my end before submitting, just wanted to highlight it.